uwreact_bringup/package.xml

```diff
@@ -17,5 +17,8 @@
   <exec_depend>frc_robot_hw</exec_depend>
   <exec_depend>frc_robot_sim</exec_depend>
   <exec_depend>zed_wrapper</exec_depend>
+  
+  <exec_depend>depthimage_to_laserscan</exec_depend>
```

Comment:
As these launch files grow, the list of dependencies here is going to get large - Let's get ahead of the curve and start putting comments. My vote is for 6 sections (But probably only 3 for now): Each subteam gets 2 sections, one for packages we write and one for 3rd party dependencies. For now, let's just do one group for each subteam, and not bother with separating 1st vs 3rd party.

Reply:
Sounds good. I'd seperate it now.